reference/waterline/models/find.md

```diff
@@ -24,6 +24,8 @@ Something.find(criteria).exec(function (err, records) {
 
 ### Example
 
+##### A basic find operation
```

Comment:
How about:

> A basic find ~~operation~~**query**


Nothing wrong with the word "operation" to describe this; just reckon we might as well try to minimize the number of different words folks have to learn